k8s/helmfile/env/production/argo-cd-base.values.yaml.gotmpl

```diff
@@ -0,0 +1,76 @@
+## Argo CD configuration
+## Ref: https://github.com/argoproj/argo-cd
+## https://raw.githubusercontent.com/argoproj/argo-helm/main/charts/argo-cd/values.yaml
+##
+
+controller:
+  resources:
+   limits:
+     cpu: 500m
+     memory: 512Mi
+   requests:
+     cpu: 250m
+     memory: 256Mi
+
+applicationSet:
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+
+dex:
+  resources:
+   limits:
+     cpu: 50m
+     memory: 64Mi
+   requests:
+     cpu: 10m
+     memory: 32Mi
+
+notifications:
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+
+redis:
+  resources:
+   limits:
+     cpu: 200m
+     memory: 128Mi
+   requests:
+     cpu: 100m
+     memory: 64Mi
+
+  exporter:
```

Comment:
is this exporter sidecar even enabled? by default I thought it was turned off?

Also if we turn it on we probably (in a follow up) want to add some Prometheus object to actually scrape it

Reply:
Ah you're right, it's disabled anyway per default